src/SourceBuild/content/eng/build.sourcebuild.targets

```diff
@@ -15,6 +15,17 @@
     <MSBuild Projects="$(RepoProjectsDir)$(RootRepo).proj" Targets="WritePrebuiltUsageData;ReportPrebuiltUsage" />
   </Target>
 
+  <!-- After building, copy SDK archive to assets root. -->
+  <Target Name="CopySdkArchive"
+          AfterTargets="Build">
+    <ItemGroup>
+      <_SdkTarball Include="$(ArtifactsAssetsDir)**/dotnet-sdk-*$(ArchiveExtension)" />
+    </ItemGroup>
+
+    <Copy SourceFiles="@(_SdkTarball)"
+          DestinationFolder="$(ArtifactsAssetsDir)" />
```

Comment:
Set `SkipUnchangedFiles` and `UseHardlinksIfPossible` so that we save disk space and so that incremental builds avoid this operation if nothing has changed.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18917/commits/9ccd9fef91977119e248048e95b7178e06691a86